package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "escape-html",
   "description": "Escape string for use in HTML",
-  "version": "1.0.3",
+  "version": "1.0.4",
```

Comment:
Please remove version bump. This is done on publish.

Reply:
oh, got it. Thank you!.
I have done rollback version.